fe1-web/src/features/evoting/network/__tests__/ElectionMessageApi.test.ts

```diff
@@ -0,0 +1,151 @@
+import { publish } from 'core/network';
+import { channelFromIds, Timestamp } from 'core/objects';
+import { SelectedBallots } from 'features/evoting/objects';
+import {
+  mockElectionNotStarted,
+  mockElectionOpened,
+} from 'features/evoting/objects/__tests__/utils';
+import 'jest-extended';
+import { mockLaoIdHash } from '__tests__/utils';
+import {
+  castVote,
+  openElection,
+  requestCreateElection,
+  terminateElection,
+} from '../ElectionMessageApi';
+import { CastVote, EndElection, SetupElection } from '../messages';
+import { OpenElection } from '../messages/OpenElection';
+
+const mockChannelId = 'some channel id';
+
+jest.mock('core/objects', () => {
+  return {
+    ...jest.requireActual('core/objects'),
+    channelFromIds: jest.fn().mockImplementation(() => mockChannelId),
+  };
+});
+
+jest.mock('core/network', () => {
+  return {
+    ...jest.requireActual('core/network'),
+    publish: jest.fn().mockImplementation(() => Promise.resolve()),
+  };
+});
```

Comment:
I believe this mock already exists [here](https://github.com/dedis/popstellar/blob/master/fe1-web/src/core/network/__mocks__/JsonRpcApi.ts).

Reply:
Oh, thank you! :)